packages/cookies/src/cookies.ts

```diff
@@ -0,0 +1,41 @@
+import type { CookieSerializeOptions } from 'cookie'
+import * as cookie from 'cookie'
+
+const normalizeCookieOptions = (options: CookieSerializeOptions) => {
+  options = Object.assign({}, options)
+
+  if (options.maxAge) {
+    options.expires = new Date(Date.now() + options.maxAge * 1000)
+  }
+
+  if (options.path == null) {
+    options.path = '/'
+  }
+
+  return options
+}
+
+const serializeValue = (value: unknown) =>
+  typeof value === 'object' ? `j:${JSON.stringify(value)}` : String(value)
```

Comment:
**question:** should we handle the case where value is an object?

If makes sense to allow booleans, strings and numbers, but objects or arrays? I'm not too sure.
There's no symetric operation, and it's not covered by tests

Reply:
I agree that it should only be a string, no reason to allow `unknown` here. But I wonder if that's making it a breaking change 🤔